src/options/quantifiers_options.toml

```diff
@@ -1608,15 +1608,6 @@ header = "options/quantifiers_options.h"
   default    = "false"
   help       = "compute backwards filtering to compute whether previous solutions are filtered based on later ones"
 
-
-[[option]]
```

Comment:
Is this related to this PR?

Reply:
Yes, this option was used for determining whether we would try to clone an ExprManager when calling subsolvers (to set optimal options on that subsolver).  Since cloning the ExprManager is no longer necessary, this option is no longer necessary.